ports/nrf/common-hal/alarm/__init__.c

```diff
@@ -48,6 +48,9 @@
 #include "nrf_power.h"
 #include "nrfx.h"
 #include "nrfx_gpiote.h"
+#ifdef NRF_DEBUG_PRINT
+void print_wakeup_cause(nrf_sleep_source_t cause);
```

Comment:
This is a fix for "no prototype" right? Instead, I think you can mark the function as `static`. That will make it clear that it is only used in this file.

Reply:
Hmm. After making it static I see that the call to print_wakeup_cause was commented out in common_hal_alarm_woken_from_sleep(). Since that could be useful debug info I made the prototype static and uncommented the function call.